### PR TITLE
reporting-operator: allow custom prometheus CA

### PR DESCRIPTION
**Pull request by tschuy**

### CODE REVIEW

File: charts/reporting-operator/templates/reporting-operator-deployment.yaml [hunk 1]
```diff
@@ -243,6 +250,10 @@ spec:
         - name: metrics-tls
           mountPath: /metrics-tls
 {{- end }}
+{{- if .Values.spec.config.prometheusCertificateAuthority.configMap.enabled }}
+        - name: {{ .Values.spec.config.prometheusCertificateAuthority.configMap.name }}
```
Comment (chancez) [inline charts/reporting-operator/templates/reporting-operator-deployment.yaml]: Name should be prometheus-certificate-authority

**Comment by tschuy** [comment]
context on the failing prow tests: I've been told that we're good to go as long as our jenkins tests work, and that we simply need to wait for the Openshift testing infra to get more reliable before we can use that as a merge blocker.

**Pull request merged**